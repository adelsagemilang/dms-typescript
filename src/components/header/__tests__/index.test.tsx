import React from 'react';
import { shallow } from 'enzyme';
import Header from 'components/header';

describe('Header Component tests', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
