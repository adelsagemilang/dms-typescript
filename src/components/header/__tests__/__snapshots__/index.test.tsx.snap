// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component tests renders correctly 1`] = `
<withHelpersModifiers(Navbar)
  id="navbar-fixed-top"
>
  <withHelpersModifiers(NavbarBrand)>
    <withHelpersModifiers(NavbarItem)
      hasDropdown={true}
      isHoverable={true}
    >
      <BackButton />
      <DropdownMenu />
    </withHelpersModifiers(NavbarItem)>
    <withHelpersModifiers(NavbarItem)>
      <img
        alt="Logan logo"
        src="/../images/logo.png"
      />
    </withHelpersModifiers(NavbarItem)>
  </withHelpersModifiers(NavbarBrand)>
  <withHelpersModifiers(NavbarMenu)>
    <withHelpersModifiers(NavbarEnd)>
      <Search />
      <Component />
      <Profile
        session={false}
      />
    </withHelpersModifiers(NavbarEnd)>
  </withHelpersModifiers(NavbarMenu)>
</withHelpersModifiers(Navbar)>
`;
